lib/order.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'menu'
+
+class Order
+    attr_reader :dishes, :order
+    
+    def initialize
+        @order = Hash.new(0)
+        @total = 0
+        @dishes = Menu.new.dishes
+        @time = (Time.now + 1*60*60).strftime("%I:%M%p")
```

Comment:
Your code totally fine as far as I can see, but you know my feelings about 12 vs 24 hour clock ;)